### PR TITLE
Bump to Build 3095 Chop Shop DLC

### PR DESCRIPTION
**Pull request by ItzCelyrian**
Added LICENSE.md (Non-Commerical 4.0)
Added 3095 - Chop Shop DLC
Added 2189 - Cayo Perico DLC
Fixed routing on Cayo Perico (Waypoints work if the player is on the island)
Fixed Minimap of Cayo Perico

TODO:
- In my servers case this wasn't necessairy but currently on Version: "3.0a" the loading of 2189 and 3095 is permanently enabled, this will need to be added to client.lua in order to allow disabling of elements.

### CODE REVIEW

**Comment by ItzCelyrian** [comment]
In this PR the following files should be updated and or ignored/adjusted.

Since I will be forking this into my own Loader I just wanted to contribute the changes I made up to 3095.

The changes which you might want to revert are found within:
- README.md (Since linkage works to my repository on my branch.)
- fxmanifest.lua (I added myself to the Authors inside my repository, you can remove this from the PR by cherry picking.)
- LICENSE.md can be scrapped, if you wish to do so.

**Review by QuirkyCarlos — APPROVED** [review]

**Review by QuirkyCarlos — APPROVED** [review]
Tested myself on my dev server literally can just drag into bob_ipl folder fine

**Comment by Virus18sp** [comment]
Will you add support for 2944 too? there are some good interiors.

**Comment by ItzCelyrian** [comment]
@Virus18sp I will create a new Version on my fork, continue tests with 2944 and then if it goes well add them to the Pull Request.👍

**Comment by ItzCelyrian** [comment]
- Added San ANdreas Mercenary (2944)
- Added Summer Special (- [Arcade])

**Comment by Sina30** [comment]
You shouldn't use it yet because it only causes problems at the casinos, and it also only causes problems near the harbor

File: README.md
[the remark this answers is not in the document]
Comment (ItzCelyrian) [reply README.md:1]: I made the Readme redirect to my Repo. That is why I recommended a Cherry-Pick merge.

**Review by TheIndra55 — CHANGES_REQUESTED** [review]
I'm sorry but this does not follow the code structure of this project at all. I'll reimplement these changes myself in another PR.

**Comment by TheIndra55** [comment]
Closing this since the latest DLCs have been added in #133.

There's still some stuff missing like arcades and cayo, but don't be hesitant to open a pull request for those. Please do follow the existing code.

**Pull request closed without merging**